python/cudf/cudf/io/orc.py

```diff
@@ -448,6 +452,7 @@ def to_orc(
                 stripe_size_bytes,
                 stripe_size_rows,
                 row_index_stride,
+                cols_as_map_type,
```

Comment:
Is it worth converting this to a `set` since it just looks to be used for membership checking?

Reply:
Converted to set 👍